dist/api_data.py

```diff
@@ -396,17 +396,17 @@ def __cmp__(self, other):
     Config('eviction', '', r'''
         eviction configuration options''',
         type='category', subconfig=[
-            Config('threads_max', '1', r'''
+            Config('threads_max', '30', r'''
```

Comment:
This part of the change makes me nervous. I really like the benefit, but setting the default limit to 30 makes me uncomfortable. Would it be reasonable to set a default of 6 or 8?